Marlin/ultralcd_impl_DOGM.h

```diff
@@ -717,21 +717,21 @@ static void lcd_implementation_status_screen() {
 
   #if ENABLED(ADVANCED_PAUSE_FEATURE)
 
-    static void lcd_implementation_hotend_status(const uint8_t row) {
+    static void lcd_implementation_hotend_status(const uint8_t row, const uint8_t extruder = active_extruder) {
```

Comment:
You can't use a global variable as the default value to a parameter.
You have to use a default like 255 instead, then do:
```cpp
const uint8_t e = extruder == 255 ? active_extruder : extruder;
```